src/components/Login/utils.ts

```diff
@@ -12,13 +12,19 @@ interface DecodedQrResponse {
 export const decodeQr = (code: string): DecodedQrResponse => {
   try {
     const parsedCode: QrCode = JSON.parse(code);
-    if (!parsedCode.endpoint) throw new Error("No endpoint specified");
-    if (!parsedCode.key) throw new Error("No key specified");
+    if (!parsedCode.endpoint)
+      throw new Error(
+        "We are currently facing login issues. Get a new QR code from your in-charge."
+      );
+    if (!parsedCode.key)
+      throw new Error(
+        "We are currently facing login issues. Get a new QR code from your in-charge."
+      );
     return { endpoint: parsedCode.endpoint, key: parsedCode.key };
   } catch (e) {
     if (e.message.includes("Unexpected token"))
       throw new Error(
-        "Invalid QR code format, consider regenerating your QR code"
+        "Scan QR code again or get a new QR code from your in-charge."
```

Comment:
Hmm, should these error messages be in the enum `ERROR_MESSAGE` as well? 
- We are currently facing login issues. Get a new QR code from your in-charge.
- Scan QR code again or get a new QR code from your in-charge.

Reply:
got it! these are added! thanks!